### PR TITLE
Update Google provider docs

### PR DESCRIPTION
**Pull request by frodenas**
Update Google provider documentation according to https://github.com/fog/fog/issues/3413.

Also I updated some outdated Google docs (supported services and valid credentials keys).

### CODE REVIEW

**Comment by frodenas** [comment]
@icco @erjohnso Can you please review this PR?

File: about/getting_started.markdown [hunk 1]
```diff
@@ -60,6 +60,12 @@ Valid keys are as follows:
     <tr><td>dnsmadeeasy_secret_key</td><td></td></tr>
     <tr><td>go_grid_api_key</td><td></td></tr>
     <tr><td>go_grid_shared_secret</td><td></td></tr>
+    <tr><td>google_project</td><td></td></tr>
```
Comment (erjohnso) [inline about/getting_started.markdown]: Description: Project ID (not number)

File: about/getting_started.markdown [hunk 2]
```diff
@@ -60,6 +60,12 @@ Valid keys are as follows:
...
+    <tr><td>google_client_email</td><td></td></tr>
```
Comment (erjohnso) [inline about/getting_started.markdown]: Description: <hash>@developer.gserviceaccount.com

File: about/getting_started.markdown [hunk 3]
```diff
@@ -60,6 +60,12 @@ Valid keys are as follows:
...
+    <tr><td>google_key_location</td><td></td></tr>
```
Comment (erjohnso) [inline about/getting_started.markdown]: Description: Path to P12 private key (not need if using the JSON file)

File: about/getting_started.markdown [hunk 4]
```diff
@@ -60,6 +60,12 @@ Valid keys are as follows:
...
+    <tr><td>google_key_string</td><td></td></tr>
```
Comment (erjohnso) [inline about/getting_started.markdown]: Description: In-line P12 private key string contents (alternative to specifying path)

File: about/getting_started.markdown [hunk 5]
```diff
@@ -60,6 +60,12 @@ Valid keys are as follows:
...
+    <tr><td>google_json_key_location</td><td></td></tr>
```
Comment (erjohnso) [inline about/getting_started.markdown]: Description: Path to JSON account file (not needed if using P12 key)

File: about/getting_started.markdown [hunk 6]
```diff
@@ -60,6 +60,12 @@ Valid keys are as follows:
...
+    <tr><td>google_json_key_string</td><td></td></tr>
```
Comment (erjohnso) [inline about/getting_started.markdown]: Description: In-line JSON private key string contents (alternative to specifying path)

File: compute/index.markdown [hunk 7]
```diff
@@ -109,14 +109,26 @@ You can run all the same ssh commands and do what you need to, then once again s
 
 ## Google Compute Engine
 
-Google has [Compute Engine](https://cloud.google.com/products/compute-engine). To get your authorization key, visit the [Google API Console](https://code.google.com/apis/console/). Once there, go to "API Access". Click "Create another client ID..." and select "service account".
+Google has [Compute Engine](https://cloud.google.com/compute/). To get your authorization key, visit the [Google Developers Console](https://console.developers.google.com). Once there, select your project, go to "APIs & auth - Credentials". Click "Create another client ID..." and select "service account".
+
```
Comment (erjohnso) [inline compute/index.markdown]: Consider adding something like: Next, download either the JSON key (preferred) or the P12 key.

And then switch the order below so the JSON instructions come before P12.

**Comment by erjohnso** [comment]
Thank you very much @frodenas! I thought it might be useful to add some text to the Description for each setting. Feel free to tweak the description text, but I don't want users to think they need to supply all of those params.

**Comment by frodenas** [comment]
@erjohnso Done.

**Comment by erjohnso** [comment]
Thanks Ferdy!

@icco - this LGTM, if you agree, can you merge?

**Comment by icco** [comment]
Man I love rich diff. Thanks! Merging.

**Comment by geemus** [comment]
Thanks!

**Pull request merged**